aderyn_nodejs/@types/aderyn_nodejs.ts

```diff
@@ -0,0 +1,22 @@
+declare module 'aderyn_nodejs' {
+
+    export function drive(
+        root: string,
+        output: string,
+        no_snippets: boolean,
+        exclude: string[],
```

Comment:
This file should help in TS linting...but don't know how to make it work without actually publishing the package